### PR TITLE
Bring installation guide in line with ecosystem docs

### PR DESCRIPTION
**Pull request by Yoshanuikabundi**
### Description

This PR links the installation guide to the new [ecosystem docs](https://docs.openforcefield.org/en/latest/install.html).

### Checklist

- [x] Lint

### CODE REVIEW

**Review by mattwthompson — APPROVED** [review]
LGTM once the upstream PR is merged

**Comment by Yoshanuikabundi** [comment]
Rendered correctly: https://openff-docs--644.org.readthedocs.build/projects/interchange/en/644/installation.html

**Pull request merged**
